test/core/iomgr/resolve_address_posix_test.cc

```diff
@@ -96,24 +95,25 @@ static void actually_poll(void* argsp) {
   grpc_millis deadline = n_sec_deadline(10);
   while (true) {
     grpc_core::ExecCtx exec_ctx;
-    {
-      grpc_core::MutexLockForGprMu lock(args->mu);
```

Comment:
can we change this to just `grpc_core::MutexLock`. Same elsewhere for this file and `test/core/iomgr/resolve_address_test.cc`

Reply:
OK, fixed